src/components/grid/index.js

```diff
@@ -127,22 +127,47 @@ Column.defaultProps = {
   pb: null,
 }
 
-const FlexColumn = styled(Column)`
-  display: block;
+const keyFrame = keyframes`
+  0% {
+    transform: scale(0);
+    opacity: 0;
+  }
+  100% {
+    transform: '';
+    opacity: '';
+  }
+`
+// styled componets V4 requeries css helper func for keyframes
+const keyFrameHelper = css`
+  animation: ${keyFrame} 360ms ease;
+`
+
+// stylelint issue req this format otherwise it errors - CssSyntaxError
+const FlexColumn = ({ animation, children, ...props }) => {
```

Comment:
This component is used in a number of other places in the codebase. Have they all been tested manually to ensure they all still work in the same manner as before the update?

Reply:
I did an initial check of all the pages but I'll do another when I've done the above comments :)